src/book/02-system/02-subsystems/04-motion.mdx

```diff
@@ -6,17 +6,68 @@ description: Motion and how it works in the NUbots codebase.
 slug: /system/subsystems/motion
 ---
 
-The motion subsystem contains modules that are responsible for walking, kicking, moving the head, and running scripts. They perform these motions by sending servo commands (called targets) to the motors.
+This page details motion modules within the NUbots main codebase, such as the walk engine and the Script Engine. Script Runner and Script Tuner are on the [Behaviour](/system/subsystems/behaviour#script-runner) page.
```

Comment:
I'd capitalise Walk Engine too, all make Script Engine lowercase to match.

```suggestion
This page details motion modules within the NUbots main codebase, such as the Walk Engine and the Script Engine. Script Runner and Script Tuner are on the [Behaviour](/system/subsystems/behaviour#script-runner) page.
```

Reply:
Yeah true, I put Script Engine uppercase because I was thinking about the modules, which are in the codebase as uppercase. But walk engine isn't a module named within the codebase. I think I'll put them all lowercase.

Reply:
```suggestion
This page details motion modules within the NUbots main codebase, such as the walk engine and the script engine. Information about modules for running and tuning scripts can be found on the [Behaviour](/system/subsystems/behaviour#script-runner) page.
```